modules/fff/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2021 Google LLC
```

Comment:
I think it's better to put this file under `zephyr/` in your repo `yperess/fff` along with `module.yml` and `Kconfig` below.

Reply:
Same as above.